docs/supportedsites.md

```diff
@@ -44,7 +44,6 @@
  - **AmericasTestKitchen**
  - **anderetijden**: npo.nl, ntr.nl, omroepwnl.nl, zapp.nl and npo3.nl
  - **AnimeOnDemand**
- - **anitube.se**
```

Comment:
this file is automatically generated.

Reply:
Thanks, I'm still in the process of getting to know the procedures here 😉.

I reverted the removal of the line in supportedsites.md. I wonder though, how the sites supported by generic extractor are listed?